### PR TITLE
LPS-121943 Test Automation: Disable by default Assigning Roles to User Segments

### PR DESCRIPTION
**Pull request by CarlosBrichete**
**Description**
This pull request is related to the story LPS-121943.

They are 2 tests included, DisableAssignRolesBySegment and EnableAssignRolesBySegment, to validate the bug.

### CODE REVIEW

**Comment by CarlosBrichete** [comment]
ci:test:sf

**Comment by CarlosBrichete** [comment]
ci:test:tango

**Comment by CarlosBrichete** [comment]
ci:test:relevant

**Comment by CarlosBrichete** [comment]
@manoelcyreno

**Comment by manoelcyreno** [comment]
LGTM

**Comment by cgoncas** [comment]
Hey @CarlosBrichete !!

Could you add a descriptive title and a description for this pull? would be awesome to be able to understand what this pull is about without the need to read the commits.


Thanks!!

**Comment by CarlosBrichete** [comment]
This pull request is related to the story [LPS-121943](https://issues.liferay.com/browse/LPS-121943).

They are 2 tests included, DisableAssignRolesBySegment and EnableAssignRolesBySegment, to validate the bug.

**Comment by CarlosBrichete** [comment]
ci:test:tango

**Comment by manoelcyreno** [comment]
ci:test:tango

**Comment by dgarciasarai** [comment]
ci:test:tango

**Comment by manoelcyreno** [comment]
ci:test:bundle

**Comment by CarlosBrichete** [comment]
ci:test:bundle

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/SegmentationAssigningRoles.testcase [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+
+	@description = "Validate if, with the option Enable Assign roles by segments disabled, the role is not working for the user"
+	@priority = "5"
+	test DisableAssignRolesBySegment {
+		property portal.acceptance = "true";
+
+		User.logoutPG();
+
+		User.loginPG(
+			password = "password",
```
Comment (manoelcyreno) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/SegmentationAssigningRoles.testcase:125]: here I think the password is "test"

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/SegmentationAssigningRoles.testcase [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+			configurationCategory = "Segments",
+			configurationName = "Segments Service",
+			configurationScope = "System Scope");
+
+		Segmentation.enableAssingRolesBySegmentsCheckbox();
+
+		User.logoutPG();
+
+		User.loginPG(
+			password = "password",
```
Comment (manoelcyreno) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/SegmentationAssigningRoles.testcase:151]: here I think the password is "test"

**Comment by CarlosBrichete** [comment]
ci:test:sf

**Comment by CarlosBrichete** [comment]
ci:test:relevant

**Comment by CarlosBrichete** [comment]
ci:test:tango

**Comment by cgoncas** [comment]
ci:forward

**Comment by cgoncas** [comment]
ci:open

**Pull request closed without merging**